sound/soc/sof/sof-audio.h

```diff
@@ -35,6 +35,9 @@
 #define SOF_DAI_CLK_INTEL_SSP_MCLK	0
 #define SOF_DAI_CLK_INTEL_SSP_BCLK	1
 
+#define SOF_ROUTE_BIND_SOURCE 0
+#define SOF_ROUTE_BIND_SINK 1
```

Comment:
still no idea what this means. A bind is between a source and a sink. I don't get what BIND_SOURCE and BIND_SINK might mean.

Reply:
I think I can update them to `SOF_ROUTE_SOURCE/SOF_ROUTE_SINK` to tell this is a source widget in a route, which is easier to understand.